cmd/zinject/translate.c

```diff
@@ -179,7 +179,7 @@ object_from_path(const char *dataset, const char *path, struct stat64 *statbuf,
 	 */
 	sync();
 
-	err = dmu_objset_own(dataset, DMU_OST_ZFS, B_TRUE, FTAG, &os);
+	err = dmu_objset_own(dataset, DMU_OST_ZFS, B_TRUE, B_TRUE, FTAG, &os);
```

Comment:
I tried to err on the side of caution here, and so anything I wasn't sure of required the keys be loaded. I will change this to `B_FALSE`.